js/src/index.js

```diff
@@ -0,0 +1,15 @@
+import './css/styles.css';
```

Comment:
Yeah, it's not useful now, but it could be later.  We could split the css into multiple files per component or use something stylus.  The import here just makes webpack load it and dump everything to `/static/styles.css`.